src/functions/Set-ItResult.ps1

```diff
@@ -117,10 +117,10 @@ function New-PesterErrorRecord {
             }).TrimEnd($([System.Environment]::NewLine))
     }
 
-    & $SafeCommands['New-Object'] Management.Automation.ErrorRecord $exception, $errorID, $errorCategory, $targetObject@{
-        exception     = $exception
-        errorId       = $errorID
-        errorCategory = $errorCategory
-        targetObject  = $targetObject
-    }
+    & $SafeCommands['New-Object'] -TypeName Management.Automation.ErrorRecord -ArgumentList @(
```

Comment:
If you modify the c# code you need to run the build.ps1 with -Clean and it will fail to run if there is a session with the dll already loaded. Then you need to close your powershell session and run it again. Annoying, but that is how it is now, the csharp code is not edited that often.

Reply:
@nohwnd I just preserved it as it was but fixed the broken hashtable, I'll look for a related example and implement it